internal/view/actions.go

```diff
@@ -76,7 +76,7 @@ func hotKeyActions(r Runner, aa ui.KeyActions) error {
 			continue
 		}
 		_, ok := aa[key]
-		if ok {
+		if ok && !hk.Override {
```

Comment:
I think we should flag this in the logs so users can check why a familiar key either stopped working or enabled a different functionality... Best to let users know which plugin/hotkey caused it.

Reply:
Hello @derailed, done for docs and logging :)